en-us/Data/Alias.xml

```diff
@@ -110,7 +110,6 @@
     <Map Name="csh-group-member-report" Link="Topics/Reports/access/group-membership-report.htm" ResolvedId="9002" />
     <Map Name="csh-healthinsight" Link="Topics/Security/healthinsight/healthinsight.htm" ResolvedId="1207" />
     <Map Name="csh-improved-ndbd" Link="Topics/Security/behavior-detection/improved-new-device-behavior-detection.htm" ResolvedId="2024" />
-    <Map Name="csh-index" Link="index.htm" ResolvedId="1298" />

```

Comment:
made the comment in the JS, but this is the line that removes it, which probably should be reverted